src/components/TodosFilter.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from 'classnames';
+import { FilterBy } from '../types/FilterOptions';
+
+type Props = {
+  filterBy: FilterBy,
+  setFilterBy: (option: FilterBy) => void;
+};
+
+export const TodosFilter: React.FC<Props> = ({ filterBy, setFilterBy }) => {
+  const setFilterByAll = (e: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
pls follow [naming convention](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)